src/main/java/org/eulu/bookshop/model/User.java

```diff
@@ -0,0 +1,29 @@
+package org.eulu.bookshop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.hibernate.validator.constraints.Length;
+
+@Data
```

Comment:
It is better not to use `@Data` with entities due to the overridden `toString()` method
```suggestion
@Getter
@Setter
```